FlowCrypt/src/test/java/com/flowcrypt/email/api/email/WkdClientTest.kt

```diff
@@ -35,4 +39,68 @@ class WkdClientTest {
     val keys = WkdClient.lookupEmail("doesnotexist@thisdomaindoesnotexist.test")
     assertTrue("Key found for non-existing email", keys == null)
   }
+
+  @Test
+  fun requestTimeoutTest() {
```

Comment:
unfortunately, it looks too complicated. Please look at [com.flowcrypt.email.rules.FlowCryptMockWebServerRule](https://github.com/FlowCrypt/flowcrypt-android/blob/8fd3efaa5b45d75a37a5d1fa30fe05d8be51e494/FlowCrypt/src/androidTest/java/com/flowcrypt/email/rules/FlowCryptMockWebServerRule.kt#L23) and [an examle of usage ](https://github.com/FlowCrypt/flowcrypt-android/blob/8e38333f14f6afeb8b26e9c4735489ac3f047112/FlowCrypt/src/androidTest/java/com/flowcrypt/email/ui/activity/enterprise/SignInActivityEnterpriseTest.kt#L75)

https://github.com/FlowCrypt/flowcrypt-android/blob/8e38333f14f6afeb8b26e9c4735489ac3f047112/FlowCrypt/src/androidTest/java/com/flowcrypt/email/ui/activity/enterprise/SignInActivityEnterpriseTest.kt#L75

https://github.com/FlowCrypt/flowcrypt-android/blob/8e38333f14f6afeb8b26e9c4735489ac3f047112/FlowCrypt/src/androidTest/java/com/flowcrypt/email/ui/activity/enterprise/SignInActivityEnterpriseTest.kt#L320

Reply:
`FlowCryptMockWebServerRule` is not available here since this is not "Android Test". However I've looked at it and it gives me idea to use okhttp MockWebServer.

Reply:
I meant that it would be better to use a mock web server instead of sockets. It will be clearer.